File: core/src/main/java/com/pluxity/park/dto/ParkUpdateRequest.java
package com.pluxity.park.dto;

import com.pluxity.facility.dto.FacilityUpdateRequest;
import io.swagger.v3.oas.annotations.media.Schema;

public record ParkUpdateRequest(
        @Schema(requiredMode = Schema.RequiredMode.REQUIRED) FacilityUpdateRequest facility,
        @Schema(requiredMode = Schema.RequiredMode.REQUIRED) String boundary) {}

File: core/src/main/java/com/pluxity/park/dto/ParkUpdateRequest.kt
package com.pluxity.park.dto

import com.pluxity.facility.dto.FacilityUpdateRequest
import io.swagger.v3.oas.annotations.media.Schema

data class ParkUpdateRequest(
    @field:Schema(requiredMode = Schema.RequiredMode.REQUIRED)
    val facility: FacilityUpdateRequest,
    @field:Schema(requiredMode = Schema.RequiredMode.REQUIRED)
    val boundary: String,
)
